internal/connector/internal/handlers/connector_admin.go

```diff
@@ -197,3 +176,205 @@ func (h *ConnectorAdminHandler) UpgradeConnectorsByOperator(writer http.Response
+	cfg := handlers.HandlerConfig{
+		Validate: []handlers.Validate{
+			handlers.Validation("namespace_id", &namespaceId, handlers.MinLen(1), handlers.MaxLen(maxConnectorNamespaceIdLength)),
+		},
+		Action: func() (i interface{}, serviceError *errors.ServiceError) {
+
+			serviceError = h.NamespaceService.Delete(request.Context(), namespaceId)
+			return nil, serviceError
+		},
+	}
+
+	handlers.HandleDelete(writer, request, &cfg, http.StatusNoContent)
+}
+
+func (h *ConnectorAdminHandler) GetNamespaceConnectors(writer http.ResponseWriter, request *http.Request) {
+	id := mux.Vars(request)["namespace_id"]
+	listArgs := coreservices.NewListArguments(request.URL.Query())
+	cfg := handlers.HandlerConfig{
+		Validate: []handlers.Validate{
+			handlers.Validation("namespace_id", &id, handlers.MinLen(1), handlers.MaxLen(maxConnectorNamespaceIdLength)),
+		},
+		Action: func() (interface{}, *errors.ServiceError) {
+
+			if len(listArgs.Search) == 0 {
+				listArgs.Search = fmt.Sprintf("namespace_id = %s", id)
+			} else {
+				listArgs.Search = fmt.Sprintf("namespace_id = %s AND (%s)", id, listArgs.Search)
+			}
+			connectors, paging, err := h.ConnectorsService.List(request.Context(), "", listArgs, "")
+			if err != nil {
+				return nil, err
+			}
+
+			result := private.ConnectorAdminViewList{
+				Kind:  "ConnectorAdminViewList",
+				Page:  int32(paging.Page),
+				Size:  int32(paging.Size),
+				Total: int32(paging.Total),
+			}
+
+			result.Items = make([]private.ConnectorAdminView, len(connectors))
+			for i, namespace := range connectors {
+				result.Items[i], err = presenters.PresentConnectorAdminView(namespace)
+				if err != nil {
+					return nil, err
+				}
+			}
+
+			return result, nil
+		},
+	}
+
+	handlers.HandleGet(writer, request, &cfg)
+}
+
+func (h *ConnectorAdminHandler) GetConnector(writer http.ResponseWriter, request *http.Request) {
+	connectorId := mux.Vars(request)["connector_id"]
+	cfg := handlers.HandlerConfig{
+		Validate: []handlers.Validate{
+			handlers.Validation("connector_id", &connectorId, handlers.MinLen(1), handlers.MaxLen(maxConnectorIdLength)),
+		},
+		Action: func() (i interface{}, serviceError *errors.ServiceError) {
+
+			connector, serviceError := h.ConnectorsService.Get(request.Context(), connectorId, "")
+			if serviceError != nil {
+				return nil, serviceError
+			}
+			return presenters.PresentConnectorAdminView(connector)
+		},
+	}
+
+	handlers.HandleGet(writer, request, &cfg)
+}
+
+func (h *ConnectorAdminHandler) DeleteConnector(writer http.ResponseWriter, request *http.Request) {
+	connectorId := mux.Vars(request)["connector_id"]
+	cfg := handlers.HandlerConfig{
+		Validate: []handlers.Validate{
+			handlers.Validation("connector_id", &connectorId, handlers.MinLen(1), handlers.MaxLen(maxConnectorIdLength)),
+		},
+		Action: func() (i interface{}, serviceError *errors.ServiceError) {
+
+			// TODO add flag to force deletion of connector and deployments
```

Comment:
Is this been implemented yet?

Reply:
flag to force deletion is a TODO

Reply:
Working on adding the force flag to nuke connector bits in the manager.

Reply:
It's not a TODO anymore!